### PR TITLE
fix: Updated python-helm-demo example to use MinIO instead of GS

### LINKED ISSUE #4690: Review Redis example
## Expected Behavior 
We should provide a validated example of using Redis as an online store.

## Current Behavior
There is one Redis example  at [Predict on an InferenceService with transformer using Feast online feature store ](https://github.com/kserve/kserve/blob/master/docs/samples/v1beta1/transformer/feast/README.md) shows an example of inference service with feast feature server using Redis on Kubernetes.
The example is outdated, and not perfectly working, according to the developer who raised the request on Slack.

## Steps to reproduce
See original message in [general](https://feastopensource.slack.com/archives/C01M2GYP0UC/p1729682339183579) Slack channel

## Possible Solution
Use the exising example in `python-helm-demo` which already has Redis as the online store, and update it properly to reduce the need of external resources (e.g. Google Cloud Storage).

### PR DESCRIPTION
**Pull request by dmartinol**
# What this PR does / why we need it:
Using MinIO instead of GCP storage to have an isolated, reproducible setup with no other external dependencies.

# Which issue(s) this PR fixes:
Fixes #4690

### CODE REVIEW

File: examples/python-helm-demo/README.md [hunk 1]
```diff
@@ -3,87 +3,168 @@
...
 For this tutorial, we set up Feast with Redis. 
 
-We use the Feast CLI to register and materialize features, and then retrieving via a Feast Python feature server deployed in Kubernetes
+We use the Feast CLI to register and materialize features from the current machine, and then retrieving via a 
+Feast Python feature server deployed in Kubernetes
 
 ## First, let's set up a Redis cluster
 1.  Start minikube (`minikube start`)
-2.  Use helm to install a default Redis cluster
+1.  Use helm to install a default Redis cluster
```
Comment (franciscojavierarceo) [inline examples/python-helm-demo/README.md:11]: Why are these numbers all 1?

File: examples/python-helm-demo/README.md [hunk 2]
```diff
@@ -3,87 +3,168 @@
...
-      sed -i '' 's/my-redis-master:6379/localhost:6379/g' feature_store.yaml
+      source minio.env
+      cd test
+      python test_python_fetch.py
       ```
-    - Then run the included fetch script, which fetches both via the HTTP endpoint and for comparison, via the Python SDK
-      ```bash
-        python test_python_fetch.py
+
+      Outpu example:
```
Comment (franciscojavierarceo) [inline examples/python-helm-demo/README.md:160]:
```suggestion
      Output example:
```

**Review by franciscojavierarceo — CHANGES_REQUESTED** [review]
small nit

File: examples/python-helm-demo/README.md (hunk 1 above)
Comment (dmartinol) [reply examples/python-helm-demo/README.md:11]: markdown replaces them with the actual sequence:
https://www.markdownguide.org/basic-syntax/#ordered-lists

**Comment by redhatHameed** [comment]
@dmartinol just confirming is there anything remaining from your side on this PR. Thanks

**Review by redhatHameed — APPROVED** [review]

**Review by franciscojavierarceo — APPROVED** [review]
lgtm

**Pull request merged**